src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -384,15 +398,15 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 				tabsContainer.classList.remove('scroll');
 
 				if (e.target === tabsContainer) {
-					this.onDrop(e, this.group.count, tabsContainer);
+					this.onDrop(e, this.tabsModel.count, tabsContainer);
 				}
 			}
 		}));
 
 		// Mouse-wheel support to switch to tabs optionally
 		this._register(addDisposableListener(tabsContainer, EventType.MOUSE_WHEEL, (e: WheelEvent) => {
-			const activeEditor = this.group.activeEditor;
-			if (!activeEditor || this.group.count < 2) {
+			const activeEditor = this.groupViewer.activeEditor;
```

Comment:
This listener feels like it should not be here but in the parent because you are looking at all editors of the group.

Reply:
I understand, however, as this is not directly related to the change of this PR, I would leave it and have a look at it as debt work